src/data/reducers/utils.test.js

```diff
@@ -22,16 +22,28 @@ describe('Reducer Utils', () => {
       expectedAssets[1].loadingFields = [assetLoading.LOCK];
       expect(addLoadingField(assets, 'chinchilla', assetLoading.LOCK)).toEqual(expectedAssets);
     });
+    it('adds loading field to existing loading fields', () => {
```

Comment:
I've added a test case to cover [this](https://github.com/edx/studio-frontend/blob/master/src/data/reducers/utils.js#L18) line. In particular, it tests the case where an asset already has an existing loadingField and the utility function is called to add another loadingField. I created a dummy loadingField called mockLoadingField to be able to add another type of loadingField to the asset, since we currently only have LOCK in constants/loadingTypes.